.travis.yml

```diff
@@ -6,43 +6,57 @@ cache:
     directories:
         - $HOME/.composer/cache
 
+env:
+  global:
+    - PHPUNIT_FLAGS="-v"
+    - ES_VERSION="5.2.1"
+
 matrix:
+  fast_finish: true
   include:
+    # Minimum supported dependencies with min and max PHP version
     - php: 5.6
-      env: SYMFONY_VERSION='^3' ES_VERSION=5.2.1
-    - php: 7.0
-      env: SYMFONY_VERSION='^3' ES_VERSION=5.2.1
-    - php: 7.1
-      env: SYMFONY_VERSION='^3' ES_VERSION=5.2.1
-    - php: 5.6
-      env: COMPOSER_FLAGS="--prefer-lowest --prefer-stable" ES_VERSION=5.2.1
+      env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
+    - php: 7.2
+      env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
+
+    # Latest supported dependencies with each PHP version
     - php: 5.6
-      env: SYMFONY_VERSION='^2' ES_VERSION=5.2.1
+    - php: 7.0
     - php: 7.1
-      env: SYMFONY_VERSION='^2' ES_VERSION=5.2.1
-  fast_finish: true
+    - php: 7.2
+      env: COVERAGE=true PHPUNIT_FLAGS="-v --coverage-clover=coverage.clover"
+
+    # Install all SF components in the same major version, see https://github.com/dunglas/symfony-lock
+    - php: 7.2
+      env: SYMFONY_VERSION="^2"
+    - php: 7.2
+      env: SYMFONY_VERSION="^3"
+    - php: 7.2
+      env: SYMFONY_VERSION="^4"
```

Comment:
that does not seem useful as it is basically the same as the `php: 7.2 env: COVERAGE=true` line